doc/api-hidden.rst

```diff
@@ -530,6 +530,12 @@
    Index.rename
    Index.copy
 
+   indexes.RangeIndex.arange
```

Comment:
Why are these hidden?

Reply:
That was to keep the "Custom Indexes" section of API Reference showing only a list of index classes. I'll add these constructors to the "creating custom indexes" subsection just below.